remotesigning/request.go

```diff
@@ -480,14 +480,15 @@ func (r *ReleaseCounterpartyPerCommitmentSecretRequest) Type() objects.RemoteSig
 // DerivationPath is the bip32 derivation path to get the key from the master key `k`.
 // Then apply MulTweak * k + AddTweak to get the final signing key.
 type SigningJob struct {
-	Id             string  `json:"id"`
-	DerivationPath string  `json:"derivation_path"`
-	Message        string  `json:"message"`
-	AddTweak       *string `json:"add_tweak"`
-	MulTweak       *string `json:"mul_tweak"`
-	Script         *string `json:"script"`
-	Transaction    *string `json:"transaction"`
-	Amount         *int64  `json:"amount"`
+	Id               string  `json:"id"`
+	DerivationPath   string  `json:"derivation_path"`
+	Message          string  `json:"message"`
+	AddTweak         *string `json:"add_tweak"`
+	MulTweak         *string `json:"mul_tweak"`
+	Script           *string `json:"script"`
+	Transaction      *string `json:"transaction"`
+	Amount           *int64  `json:"amount"`
+	L1DerivationPath string  `json:"l1_derivation_path"`
```

Comment:
Can we reuse the DerivationPath field? The signing job is either a L1 job or a L2 job so there should be a field to identify if it's L1 or L2, but use the same derivation path field.

Reply:
cc @alecchendev do you know who's working on the sparkcore changes?

Reply:
So the issue here is that the L1_derivation _path is always changing between transactions. So we need to take the hardened part of the DerivationPath field and then the nonhardened part of the L1DerivationPath field. 

I think a workaround for this if we want to avoid adding the new field in signable_payload is combining the derivation paths in l1_wallet before we pass it into the go-sdk. I think that probably works better actually. Will change this.

Reply:
oh, so basically the derivation path is something like m/84'/3' then the L1 path is the children after it? 

This is pretty confusing tbh... I feel like we should provide a cleaner way to do this. Since this is an example implementation for customers. This will create a lot of customer support work afterwards.

Reply:
Yeah I think it's better to just combine the path in a single field here.

Reply:
Yeah exactly, took @alecchendev and I awhile to figure this out. But this is the only way that we can re-derive the Tx script from the master seed. Is there another way we can validate the script?

Reply:
I will definitely combine them to one field. Makes it way simpler since I don't have to do alembic and add a field into signable payload as well. Would this still be too confusing for customers?

Reply:
Yeah combining it should be fine.

Reply:
so the `DerivationPath` is for the key that we are using to sign to spend the previous output, and the `L1DerivationPath` is for the key that we're sending to in the change output, which will always be different. So i don't really know how we could combine these two.

Reply:
I am combining them right before we put the derivation path in the payload. That way we still validate its an L1 transaction with the m/84' prefix and still use the L1DerivationPath to get the script. I think this works as expected.

Reply:
talked in person about keeping them separate but adding docs